### PR TITLE
[#895] assets

### PR DESCRIPTION
**Pull request by kardan**
Organised included assets

### CODE REVIEW

File: akvo/rsr/views/my_rsr.py [hunk 1]
```diff
@@ -29,7 +29,7 @@ def my_details(request):
 
     profile_form = ProfileForm(
         initial={
-            'email': request.user.email,
+            'emailform': request.user.email,
```
Comment (KasperBrandt) [inline akvo/rsr/views/my_rsr.py]: Why this change?

Comment (kardan) [inline akvo/rsr/views/my_rsr.py]: Might be a mistake on my part. Hade to debug and later realized it was in the Ajax url. Will review when I can

Comment (KasperBrandt) [inline akvo/rsr/views/my_rsr.py]: Easy to fix though; just name it back to 'email' and all works again :)..
